runtime/compiler/control/J9Options.cpp

```diff
@@ -2468,6 +2477,25 @@ bool J9::Options::feLatePostProcess(void * base, TR::OptionSet * optionSet)
       }
 #endif
 
+#if defined(J9VM_OPT_JITSERVER)
+   // Change the values of the two thresholds if the user didn't
+   // input values to the options.
+   if (_veryHighActiveThreadThreshold == -1)
+      veryHighActiveThreadThreshold = getNumUsableCompilationThreads() * 0.9;
+  
+   if (_highActiveThreadThreshold == -1)
+      _highActiveThreadThreshold = getNumUsableCompilationThreads() * 0.8;
+
+   // Check if the user inputted the correct values.
+   // If the user entered wrong values, return thresholds to default values.
+   if (_veryHighActiveThreadThreshold > getNumUsableCompilationThreads() || _veryHighActiveThreadThreshold < _highActiveThreadThreshold)
```

Comment:
I am thinking that if the thresholds are bigger than the usableCompilationThreads we should set then to be equal to the number of compilation threads. So, the code becomes:
```
if (_veryHighActiveThreadThreshold > getNumUsableCompilationThreads())
   _veryHighActiveThreadThreshold = getNumUsableCompilationThreads();
if (_highActiveThreadThreshold > getNumUsableCompilationThreads())
   _highActiveThreadThreshold = getNumUsableCompilationThreads();
if (_highActiveThreadThreshold  > _veryHighActiveThreadThreshold)
   _highActiveThreadThreshold  = _veryHighActiveThreadThreshold
```